lib/peep.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+class Peep
+
+  def self.all
+    if ENV['RACK_ENV'] == 'test'
```

Comment:
Also, single responsibility principle: I feel like this should be a separate method get_connection, or event a separate class, e.g.:
``` 
connection = Connection.new_connection();
```
or
``` 
connection = Database.connect();
```
I notice that you've repeated this code below, which suggests that it could be refactored to avoid duplication.